### PR TITLE
Move Veertu Desktop to non-versioned download URL

### PR DESCRIPTION
**Pull request by dwaite**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Thank you, but we prefer versioned downloads. Our users overwhelmingly chose it that way.

**Comment by miccal** [comment]
The current download link `ttps://d2sje6b9huarvp.cloudfront.net/VeertuDesktop-#{version}.dmg` no longer works though.

**Comment by vitorgalvao** [comment]
> The current download link `ttps://d2sje6b9huarvp.cloudfront.net/VeertuDesktop-#{version}.dmg` no longer works though.

Ah, that’s a different matter, then.

**Review by vitorgalvao — APPROVED** [review]

**Comment by jawshooah** [comment]
Alternatively, we could keep the version number and add an appcast: https://d2sje6b9huarvp.cloudfront.net/vdoaupd.rss

**Comment by vitorgalvao** [comment]
> Alternatively, we could keep the version number and add an appcast: https://d2sje6b9huarvp.cloudfront.net/vdoaupd.rss

Yes, best solution.

@dwaite Mind adding the [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

**Comment by dwaite** [comment]
will do.

File: Casks/veertu-desktop.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 cask 'veertu-desktop' do
-  version '1.2.87'
-  sha256 'e5edb743e6ba9f84b98475e140b978679a6fd6b9c2cdb63c29e736780670a6c7'
+  version '1.2.83'
```
Comment (vitorgalvao) [inline Casks/veertu-desktop.rb]: Wait, why did we drop a version? According to the appcast we can even go higher than before, to `1.2.90`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by adidalal** [comment]
Versioned download seems to work, actually.
https://github.com/caskroom/homebrew-cask/pull/26443

**Pull request closed without merging**